From 0a612bab30c89e5e1f566e2bed2671fdc20f094c Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 24 Jul 2019 22:39:25 +0200
Subject: [PATCH 363/386] mmc: sdhic-xenon: remove redundant assignment

The voltage_switch pointer is already assigned at the beginning of
sdhci_xenon_ops - remove redundant entry.

Change-Id: Ie5f19d973fa3257ccdd7dfa450de3525c760a402
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/13021
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Marcin Wojtas <marcin@marvell.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/mmc/host/sdhci-xenon.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci-xenon.c b/drivers/mmc/host/sdhci-xenon.c
index e27ad13fe026..2a5e90698613 100644
--- a/drivers/mmc/host/sdhci-xenon.c
+++ b/drivers/mmc/host/sdhci-xenon.c
@@ -254,7 +254,6 @@ static const struct sdhci_ops sdhci_xenon_ops = {
 	.reset			= xenon_reset,
 	.set_uhs_signaling	= xenon_set_uhs_signaling,
 	.get_max_clock		= sdhci_pltfm_clk_get_max_clock,
-	.voltage_switch		= xenon_voltage_switch,
 };
 
 static const struct sdhci_pltfm_data sdhci_xenon_pdata = {
-- 
2.17.1

